README.md

```diff
@@ -32,36 +48,52 @@ Navigate to http://localhost:3000/admin and enter your password when prompted.
 Select the “Initialize Database” button and wait for the database to populate.
 Once this process completes, the application is ready to use!
```

Comment:
This section is outdated. Guide on how to update it would be appreciated.

Reply:
That button was busted *before* we merged `meteor-shower`. It was on our to-do list for backend fixes. Right now the only way to populate the database is with mongorestore.

Reply:
Something like this dumprestore.sh which restores relevant bson files passed to it from a directory to a local mongodb running at port 3001.
```
# Restore all bson files found in the directory passed in as a command line argument
mongorestore -h 127.0.0.1 --port 3001 -d test "$1/classes.bson" --drop
mongorestore -h 127.0.0.1 --port 3001 -d test "$1/subjects.bson" --drop
mongorestore -h 127.0.0.1 --port 3001 -d test "$1/reviews.bson" --drop
```

Reply:
@khemritolya since you are more knowledge on this, could you help fix this part of the docs?